test/index.js

```diff
@@ -755,6 +762,11 @@ describe('PaymentProtocol', function() {
       trust.caName.should.equal('Go Daddy Class 2 CA');
       trust.chainVerified.should.equal(true);
 
+      // Verify that expiration will fail verification
+      clock.restore();
+      var verified2 = pr.x509Verify();
```

Comment:
Just for clarity, we should either manually stub the clock for some time far in the future or make a comment explaining that the expiration date on the cert is april 2016.